src/templates/pt/2019/base_chapter.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base/2019/base_chapter.html" %}
+
+{% block title %}{{ metadata.get('title') }} | 2019 | Web Almanac por HTTP Archive{% endblock %}
+
+{% block description %}{{ metadata.get('description',metadata.get('title') + ' capítulo do ' + year + ' Web Almanac sondando o uso de ' + metadata.get('description',metadata.get('title')) + ' na web.') }}{% endblock %}
+
+{% block twitter_image_alt %}Imagem do capítulo {{ metadata.get('title') }} capítulo de {{ year }} Web Almanac{% endblock %}
+
+{% block unedited %}[Não editado]{% endblock %}
+
+{% block prev_next_title %}Navegação do capítulo anterior e seguinte{% endblock %}
+
+{% block discuss_this_chapter %}Discuta este capítulo{% endblock %}
+{% block comment %}Comente{% endblock %}
+{% block comments %}comentários{% endblock %}
+{% block queries %}Ver consultas{% endblock %}
+{% block results %}Ver resultados{% endblock %}
+
+{% block previous %}Capítulo anterior (pressione 'p' ou ','){% endblock %}
+{% block next %}Próximo Capítulo (pressione 'n' ou '.'){% endblock %}
```

Comment:
That telltale red mark shows we need a new line at the end of this file (some programs complain about missing new lines hence best practice to always end a text file with one - which is why GitHub highlights it when you don't).

```suggestion
{% block next %}Próximo Capítulo (pressione 'n' ou '.'){% endblock %}

```